attest/verifier/config/src/lib.rs

```diff
@@ -128,29 +171,24 @@ impl core::fmt::Display for TrustedMeasurementSet {
 }
 
 impl TrustedMeasurementSet {
-    /// Create a verifier from this measurement set for a given enclave name.
-    pub fn create_verifier(&self, enclave_name: impl AsRef<str>) -> Result<Verifier, Error> {
+    /// Get the measurements for a given enclave name.
+    pub fn measurements(
+        &self,
+        enclave_name: impl AsRef<str>,
+    ) -> Result<Vec<TrustedMeasurement>, Error> {
         let enclave_name = enclave_name.as_ref();
-        let mut count = 0usize;
-
-        let mut verifier = Verifier::default();
-        verifier.debug(DEBUG_ENCLAVE);
-
-        for measurements in self.table.values() {
-            if let Some(measurement) = measurements.get(enclave_name) {
-                // TODO: it would be nice to add the release name in also as like
-                // a debug string somewhere that would show up in error messages
-                // when attestation fails?
-                measurement.add_to_verifier(&mut verifier);
-                count += 1;
+        let mut measurements = vec![];
+
+        for row in self.table.values() {
+            if let Some(measurement) = row.get(enclave_name) {
+                measurements.push(measurement.clone());
             }
         }
 
-        if count == 0 {
-            return Err(Error::NoMeasurementsFound(enclave_name.to_owned()));
+        match measurements.len() {
+            0 => Err(Error::NoMeasurementsFound(enclave_name.to_string())),
+            _ => Ok(measurements),
```

Comment:
IFF this is the path we want to take, wasn't sure if we still wanted a result or if an empty vec would suffice. 

For this use case, I like the result so consumers of the API can get stopped right here instead of passing an empty measurement vec down to the verifier which will(should) reject it.